tests/templates/test_subroutines/test_fable.py

```diff
@@ -59,17 +56,10 @@ def test_standard_validity(self, input_matrix):
                 ),
                 5,
             ),
-            (
-                np.random.random((8, 8)),
-                7,
-            ),
-            (
-                np.random.random((16, 16)),
-                9,
-            ),
```

Comment:
This test case is being removed.

Reply:
Yes exactly. @austingmhuang will also have a look to make sure it's okay to remove :)

Reply:
Yes I think it's fine to remove this test.